src/api/java/com/minecolonies/api/advancements/AdvancementTriggers.java

```diff
@@ -37,7 +37,7 @@
     public static final DeepMineTrigger             DEEP_MINE              = new DeepMineTrigger();
     public static final AllTowersTrigger            ALL_TOWERS             = new AllTowersTrigger();
     public static final CitizenBuryTrigger          CITIZEN_BURY           = new CitizenBuryTrigger();
-    public static final CitizenResurrectrigger      CITIZEN_RESURRECT      = new CitizenResurrectrigger();
+    public static final CitizenResurrecTrigger CITIZEN_RESURRECT      = new CitizenResurrecTrigger();
```

Comment:
Typo in the name.

Reply:
That's outdated, was fixed already (unless they missed a spot)